examples/with-mdx/src/pages/index.mdx

```diff
@@ -14,4 +14,4 @@ Written by: {new Intl.ListFormat('en').format(authors.map(d => d.name))}.
 
 Published on: {new Intl.DateTimeFormat('en', {dateStyle: 'long'}).format(published)}.
 
-<Counter client:idle>## This is a counter!</Counter>
```

Comment:
Remove `client:` directive. This is causing an infinite build loop that our smoke test (thankfully!) picked up